### DISCUSSION TITLE
Option flatten vs compact

### DISCUSSION THREAD

**Discussion by Hurtak** [gcanti/fp-ts#1794] [Q&A] [answered]
I am checking https://gcanti.github.io/fp-ts/modules/Option.ts.html#compact and https://gcanti.github.io/fp-ts/modules/Option.ts.html#flatten

They both have the same type signature, are they just aliases or do they do something different under the hood?

**Comment by gcanti** [comment, marked as answer]
Aliases https://github.com/gcanti/fp-ts/blob/435fe155ac898980734b2de6600151cecf099a6b/src/Option.ts#L929
